modules/editor/defaultEditorManagerState.ts

```diff
@@ -17,6 +17,11 @@ export const DEFAULT_EDITOR_MANAGER_STATE: SnapshotIn<typeof EditorManager> = {
             value: 1,
           },
         },
+      ],
+    },
+    {
+      content: "Now with multimessage support\n_ _",
+      embeds: [
```

Comment:
This disrupts reading flow, please revert as I'll need to update the entire thing some day anyway